x-pack/plugin/sql/sql-proto/src/main/java/org/elasticsearch/xpack/sql/proto/ColumnInfo.java

```diff
@@ -27,40 +27,50 @@
     private static final ConstructingObjectParser<ColumnInfo, Void> PARSER =
         new ConstructingObjectParser<>("column_info", true, objects ->
             new ColumnInfo(
-                objects[0] == null ? "" : (String) objects[0],
+                (String) objects[0],
```

Comment:
this is breaking change and requires backwards compatibility checks.

Reply:
> requires backwards compatibility checks

I can revert and adapt around it if needed, but I couldn't  found a place where we actually expose the `table`.